Practice/Practice/Controllers/EmployeeController.cs

```diff
@@ -27,13 +27,14 @@ public async Task<ActionResult> Index()
         public async Task<ActionResult> Delete(int? id)
         {
 
-            if(!id.HasValue)
+            if (!id.HasValue)
             {
                 return RedirectToAction("Index");
             }
 
             await repository.DeleteEmployee(id.Value);
-
+            return RedirectToAction("Index");
```

Comment:
Fajnie jakby to jednak było obsłużone w AJAX'ie i metoda zwracała JSON'a z rezultatem działania, który byłby wyświetlony, np. w toastr użytkownikowi